### ISSUE TITLE
Make it compatible with luatex > 0.85

### ISSUE THREAD

**Issue by nichtleiter** [yihui/tikzDevice#150]
For luatex > 0.85 

```
tikzLualatexPackages = c
(
"\\usepackage{tikz}\n",
"\\usepackage[active,tightpage,psfixbb]{preview}\n",
"\\usepackage{fontspec,xunicode}\n",
"\\PreviewEnvironment{pgfpicture}\n",
"\\setlength\\PreviewBorder{0pt}\n"
)
```

doesn't work anylonger.

To get appropriate results it must be set to 

```
tikzLualatexPackages = c
(
"\\usepackage{tikz}\n",
"\\usepackage{luatex85}\n",
"\\usepackage[active,tightpage]{preview}\n",
"\\usepackage{fontspec,xunicode}\n",
"\\PreviewEnvironment{pgfpicture}\n",
"\\setlength\\PreviewBorder{0pt}\n"
)
```

It would be great if luatex > 0.85 is also considered in future developments...

Best regards and thanks for the great effort you put into tikzDevice!

**Comment by krlmlr** [comment]
Thanks for your input. I'm not sure about the best way to incorporate this -- will `\usepackage{luatex85}` also work with LuaTeX <= 0.85? Happy to review a pull request.

**Comment by krlmlr** [comment]
Also, from the luatex85 documentation:

> Note that if packages are found that require luatex85 you may want to contact
the authors asking that the packages be updated to current LuaTEX syntax. The
luatex85 package should be seen as a temporary aid to improve compatibility
during the transition towards LuaTEX 1.0 it is not intended that future documents
should always have to load this compatibility emulation.

I'm also missing a reproducible example that shows what exactly doesn't work with current LuaLaTeX.

**Comment by krlmlr** [comment]
Thanks, I've adapted your suggestion, because I was seeing test failures. Please open a new issue in case of any problems.

**Comment by EmmanuelCharpentier** [comment]
I can confirm that, with the current LuaLaTex (1.0.4) : 
* the version of `tikzDevice` available on CRAN (0.10.1, IRC) is faulty ;
* the fix proposed in this issue is insufficient ;
* the version (0.10-5) available on GitHub works correctly.

Shouldn't this latter version be released on CRAN ?

**Comment by krlmlr** [comment]
Thanks. Working on a CRAN release.
